insights_content_template_renderer_tests.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -x
+
+# Copyright 2022, 2023 Red Hat, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+dir_path=$(dirname "$(realpath "$0")")
+export PATH=$PATH:$dir_path
+export PATH_TO_LOCAL_TEMPLATE_RENDERER=${PATH_TO_LOCAL_TEMPLATE_RENDERER:="../insights-content-template-renderer"}
+
+#set NOVENV is current environment is not a python virtual env
+[ "$VIRTUAL_ENV" != "" ] || NOVENV=1
+
+function install_reqs() {
+    python3 "$(which pip3)" install -r requirements.txt
+    python3 "$(which pip3)" install -r requirements/insights_content_template_renderer.txt
+}
+
+function prepare_venv() {
+    echo "Preparing environment"
+    # shellcheck disable=SC1091
+    virtualenv -p python3 venv && source venv/bin/activate
+    echo "Environment ready"
+}
+
+function install_service() {
+    cd "$PATH_TO_LOCAL_TEMPLATE_RENDERER"
```

Comment:
you should clone it if not there. something like https://github.com/RedHatInsights/insights-behavioral-spec/blob/438bbe1ba59f04e18b76aabebb30238b1ecac2f9/insights_content_service_test.sh#L54